src/target/riscv/riscv-013.c

```diff
@@ -4088,7 +4085,8 @@ static int riscv013_get_register(struct target *target,
 		uint64_t dcsr;
 		/* TODO: move this into riscv.c. */
 		result = register_read(target, &dcsr, GDB_REGNO_DCSR);
-		*value = get_field(dcsr, CSR_DCSR_PRV);
+		*value = set_field(0, VIRT_PRIV_V, get_field(dcsr, CSR_DCSR_V));
```

Comment:
Ah, so "V" is a new DCSR bit in the draft debug spec, and "priv" is a pseudo register abstraction with just the privilege mode and virtualization mode from DCSR.